src/elements.rs

```diff
@@ -139,12 +194,33 @@ impl Element {
         let cmd = WebDriverCommand::GetElementProperty(self.element.clone(), prop.to_string());
         match self.client.issue(cmd).await? {
             Json::String(v) => Ok(Some(v)),
+            Json::Bool(b) => {
+                // NOTE: boolean properties such as "checked" seem to be returned as boolean
+                //       values rather than strings.
+                Ok(Some(b.to_string()))
+            }
             Json::Null => Ok(None),
             v => Err(error::CmdError::NotW3C(v)),
         }
     }
 
-    /// Retrieve the text contents of this elment.
+    /// Look up a CSS property for this element by name.
+    ///
+    /// `Ok(String::new())` is returned if the the given CSS property is not found.
```

Comment:
Why not use `Option` here same as for `prop` above?

Reply:
It's tricky. The spec is different for this one, and it probably never actually returns `null`. 

```
Let computed value be the result of the first matching condition:

current browsing context’s document type is not "xml"
    computed value of parameter property name from element’s style declarations. property name is obtained from url variables. 
Otherwise
    "" (empty string) 
```

I noticed this when testing `thirtyfour`. If you query a CSS property that doesn't exist, you don't get null/None. You get an empty string.

I don't really like the idea of changing it to `Option<String>` if it never actually returns `None` in practice. However the alternatives are to leave it like this, or otherwise remove the `Json::Null` variant and let that fall through to `NotW3C`.

What's your preference?

Reply:
Interesting! In that case, yeah, let's leave the return type as-is, but change `Null` (which it sounds like can't be returned) to be `NotW3C` instead.

Reply:
Should this be called `css_prop` instead?

Reply:
I see you renamed it to `css_value`, which seems reasonable :+1: